internal/pkg/auth/sso/auth_state.go

```diff
@@ -63,6 +75,8 @@ type authState struct {
 	SSOProviderClientID           string
 	SSOProviderCallbackUrl        string
 	SSOProviderIdentifier         string
+	SSOProviderScope              string
+	SSOProviderIDP                string
```

Comment:
I see the IDP being set, but I'm not sure where it's used.

Reply:
It's defined here: https://github.com/confluentinc/cli/pull/1871/files#diff-796fc20ed244a970061f37913241dbb8c512cfff8a27fbc3391f9b1bccbe3035R36
and used here: https://github.com/confluentinc/cli/pull/1871/files#diff-796fc20ed244a970061f37913241dbb8c512cfff8a27fbc3391f9b1bccbe3035R115